.gitignore

```diff
@@ -1 +1,3 @@
-build*/
\ No newline at end of file
+build*/
```

Comment:
https://mesonbuild.com/FAQ.html#how-do-i-ignore-the-build-directory-in-my-vcs

You could probably get rid of this now.